File: esercizi_java/test/compito_python_3/Nutrizionista.py
from Dipendenti import Dipendenti

class Nutrizionista(Dipendenti):
    def __init__(self, cod, nome, costo, telefono, medico, appuntamenti):
        super().__init__(cod, nome, costo)
        self.telefono = telefono
        self.medico = medico
        self.appuntamenti = appuntamenti
        self.costo = costo

    def toString(self):
        return super().toString() + "\tnutri"+"\t-\t-\t-\t" + str(self.telefono) + "\t"+str(self.medico)+"\t"+str(self.appuntamenti)+"\t"+str(self.costo)

File: esercizi_java/NetBeansProjects/Palestra/src/palestra/Nutrizionista.java
package palestra;

/**
 *
 * @author tobiasacchetto
 */
public class Nutrizionista extends Dipendente{
    int appuntamenti;
    boolean medico;
    long telefono;
    
    public Nutrizionista(int cod, String nome, double cost, long telefono, boolean medico, int appuntamenti){
        super(cod,nome,cost);
        this.telefono=telefono;
        this.medico= medico;
        this.appuntamenti = appuntamenti;
    }
    
    public String toString(){
        return super.toString()+"\tnutri"+"\t-\t-\t"+telefono+"\t"+medico+"\t"+appuntamenti+"\t"+super.cost;
    }
}
